### DISCUSSION TITLE
In the next version, can you add a product-based separate payment system to the restaurants, urgently needed.

### DISCUSSION THREAD

**Discussion by lim10soft** [Blair2004/NexoPOS#1125] [Gastro 4.x Ideas & Issues]
In the next version, can you add a product-based separate payment system to the restaurants, urgently needed.

https://user-images.githubusercontent.com/102427740/182651626-6fe02892-dfa8-42de-b029-3d9ee82f03ba.png

**Comment by Blair2004** [comment]
Hi, unfortunately I don't think it's something we want to see on the core at the moment.

We however value your contribution and are thankful for this.

Kind regards.

**Comment by lim10soft** [comment]
We are having a hard time managing the restaurant. We have to go back and
look at the price of the product to get paid separately from customers.

Blair Jersyer ***@***.***>, 23 Eki 2022 Paz, 19:59 tarihinde
şunu yazdı:

**Reply by nexopos** [reply]
You can alternatively split each product you want to pay separately and make the payment.
We cannot create sub order on an order.

**Reply by lim10soft** [reply]
no restaurant owner can deal with this among so many customers, please try to come up with different easy solutions, we are really having a hard time. You have to use a calculator.

**Reply by Blair2004** [reply]
Sorry after further thoughts, this is not something we can merge.
